lib/streamlit/hello.py

```diff
@@ -17,55 +17,172 @@
 
 import streamlit as st
 import numpy as np
-import time
+import inspect
+from collections import OrderedDict
 
 
-def run():
-    st.title("Hello Streamlit!")
-    st.write(
+def intro():
+    st.markdown(
         """
-        This is an example **Streamlit** app.
+## Intro...
+
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+"""
+    )
 
-        If this is your first time using Streamlit, welcome! You may want to
-        take a look at our
-        [tutorials and documentation](https://streamlit.io/secret/docs) next.
-        And feel free to ask us questions any time at
-        [help@streamlit.io](mailto:help@streamlit.io).
 
-        To celebrate this magic moment, below we're generating a bunch of
-        random numbers in a loop for around 10 seconds. Enjoy!
+def demo_1():
     """
-    )
+    This demo shows how you can use Streamlit to format text in a few different
+    ways.
+    """
+    st.title("✨✨✨ Hello 🌍!!!! ✨✨✨")
+    st.header("This is a header...")
+    st.subheader("... and this is a subheader")
+    st.write("We can write down something...")
```

Comment:
maybe change `We can write down something...` to `We can write something...`

Reply:
yeah... this dramatically changed